Raven.NET.Analytics/Misc/RandomDataGenerator.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Raven.NET.Analytics.Model;
+using Raven.NET.Core.Subjects;
+
+namespace Raven.NET.Analytics.Misc
+{
+    public static class RandomDataGenerator
```

Comment:
Should random data generation be in the analytics project? Not like somewhere in the examples?

Reply:
As discussed via DM, as this is alfa version still lets keep it there